src/inverseRobot.js

```diff
@@ -24,7 +24,17 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  let obj = {};
+
+  for (const key in robot) {
+    obj[robot[key]] = key;
+  }
```

Comment:
Let's add if statement here for checking if [robot[key]] already exist in obj, it will be more readable and easier for understanding code logic